stockflux-chart/src/index.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import '../node_modules/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.css';
+
+import 'jquery/src/jquery';
+import 'malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.js';
```

Comment:
See above, but also seems to imported twice, can we remove all of these excet the react imports?